doc/ADVANCED.md

```diff
@@ -31,6 +31,9 @@ In order to retrigger a specific test, you need to add a comment to the PR.
 **PLESE NOTE**: gitarro will delete the comment where you write the retriggering command,
 so make sure that you use a separate comment for this.
 
+**NEW!**: You can also create a checkbox in the PR description with the text : Re-run test "<test_name>", 
+if it's checked gitarro will re-run the test and uncheck it.
```

Comment:
this documentation is to much simple. 

I would add a concrete example of the feature, so it's more obvious. :sunflower:

Reply:
Yeah.. I let it that for the end, doc must be improved and probably a bit refactored at that section, agree.
Most important part for me is that we agree in code :)